omod/src/main/webapp/pages/visit/templates/sections/hivPlanSectionHeading.gsp

```diff
@@ -4,9 +4,5 @@
 </span> <!-- encounter-type class added for smoke tests -->
 <span class="obs-span">
 
-    <span ng-show="activeDrugOrders.\$resolved && activeDrugOrders.length > 0">
-        {{ activeOrderConcepts() | omrsDisplay }}
-    </span>
```

Comment:
meant for this to go in a different branch, sorry... but just removing this since it doesn't work to avoid confusion down the line...